caffe2/operators/slice_op.h

```diff
@@ -198,22 +198,26 @@ bool SliceImpl(
 
 } // namespace
 
-template <class SIndex, class Context>
+template <class Context>
 class SliceOp : public Operator<Context> {
  public:
   USE_OPERATOR_CONTEXT_FUNCTIONS;
   SliceOp(const OperatorDef& operator_def, Workspace* ws)
       : Operator<Context>(operator_def, ws),
-        starts_(this->template GetRepeatedArgument<SIndex>("starts")),
-        ends_(this->template GetRepeatedArgument<SIndex>("ends")),
+        starts_(this->template GetRepeatedArgument<TIndex>("starts")),
```

Comment:
This seems a breaking changes. We may have models in which `Slice` may take int32 tensor as inputs, with this change, they may break.

Reply:
Note for anyone reading, this is actually not a breaking change since in the Proto the attributes are stored as int64 and this is doing a narrowing conversion under the hood